plugins/system/jacademyseo_opengraph/language/en-GB/plg_system_jacademyseo_opengraph.sys.ini

```
@@ -0,0 +1,16 @@
+; JacademySEO Open Graph Plugin System Language File
+; Copyright (C) 2025 JacademySEO. All rights reserved.
+; License GNU General Public License version 2 or later
+
+; Plugin Name and Description for Extension Manager
+PLG_SYSTEM_JACADEMYSEO_OPENGRAPH="System - JacademySEO Open Graph"
```

As the plugin should go directly into the core, please adjust all the language strings and remove JACADEMYSEO
```suggestion
PLG_SYSTEM_OPENGRAPH="System - Open Graph"
```